dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/Admin/MergeUser/ChooseTrn.cshtml.cs

```diff
@@ -0,0 +1,70 @@
+using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.EntityFrameworkCore;
+using TeacherIdentity.AuthServer.Models;
+
+namespace TeacherIdentity.AuthServer.Pages.Admin.MergeUser;
+
+public class ChooseTrn : PageModel
+{
+    public new User? User { get; set; }
```

Comment:
Nit: could we store just the properties we need here instead of the whole model objects?